.dockerignore

```diff
@@ -1,6 +1,4 @@
-packages/*
-.git/*
-.github/*
-.vs/*
-.nuget/*
-Tests/*
\ No newline at end of file
+.git/
+.github/
+.vs/
+.nuget/*
```

Comment:
Wonder if removing `packages` and `Tests` from being ignored is a required change?

Reply:
Whoops was an early change but is no longer required. I will undo them.

Reply:
Done!